Assets/LeapMotion/Scripts/Attributes/MaxValue.cs

```diff
@@ -19,13 +19,19 @@ public class MaxValue : CombinablePropertyAttribute, IPropertyConstrainer {
         property.floatValue = Mathf.Min(maxValue, property.floatValue);
       } else if (property.propertyType == SerializedPropertyType.Integer) {
         property.intValue = Mathf.Min((int)maxValue, property.intValue);
-      }
+      } else if (property.propertyType == SerializedPropertyType.Vector2) {
+        property.vector2Value = new Vector2(Mathf.Min(minValue, property.vector2Value.x), Mathf.Min(minValue, property.vector2Value.y));
```

Comment:
`property.vector2Value = Vector2.Min(property.vector2Value, Vector2.one * minValue);`
Is a little bit smaller, and can be more easily expanded to Vector3 (and why no love for Vector4? D:)